### PR TITLE
html, handlebars. routes changes

### PR DESCRIPTION
**Pull request by tatiana404**

### CODE REVIEW

File: controllers/home-routes.js
Comment (aedelfrid) [inline controllers/home-routes.js:1]: uncomment the code in controllers/home-routes.js

File: views/homepage.handlebars
Comment (aedelfrid) [inline views/homepage.handlebars:1]: there appears to be an issue with the page that shows the body being inserted twice

**Review by aedelfrid — CHANGES_REQUESTED** [review]

File: views/homepage.handlebars
Comment (secretariuss) [reply views/homepage.handlebars:1]: I think we need to delete from 27 row  header to 66 row script... in views/layouts/main.handlebars  and  add link rel="stylesheet" type="text/css" href="/css/login.css" on 12 row

Comment (aedelfrid) [reply views/homepage.handlebars:1]: that part is fine I think. there is a stray {{{body}}} in views/layouts/main.handlebars at line 21. at least in the working copy

**Review by aedelfrid — APPROVED** [review]

**Review by SkylerKeeling — APPROVED** [review]

**Pull request merged**